common/source/docs/common-beagle-bone-blue.rst

```diff
@@ -287,7 +287,6 @@ use either the stable tag ``ArduPlane-stable`` or the current Plane branch ``Pla
 
     ./Tools/environment_install/install-prereqs-ubuntu.sh
     git checkout Plane-4.1
-    git submodule update --init --recursive
```

Comment:
This doesn't look right; you've jsut checked out an older version of the software, a submodule update is often required,

Reply:
Proposal: init once, during clone. Then, just call update each time you change branches in this guide.

Reply:
> Proposal: init once, during clone. Then, just call update each time you change branches in this guide.

I think that flow would break if we add a new submodule.

Pretty sure the `--init` is essentially free anyway.

I *do* like the premise of this PR - when cloning add the extra argument.   I *think* `--recurse` may be sufficient vs `--recurse-submodules`... less typing?

Reply:
> > Proposal: init once, during clone. Then, just call update each time you change branches in this guide.
> 
> I think that flow would break if we add a new submodule.
> 
> Pretty sure the `--init` is essentially free anyway.
> 
> I _do_ like the premise of this PR - when cloning add the extra argument. I _think_ `--recurse` may be sufficient vs `--recurse-submodules`... less typing?

Yes, I can add `--init` in other areas as needed in the guide. 

The `--recurse` option is not available for me under the `git clone --help` listing.

Reply:
> > Proposal: init once, during clone. Then, just call update each time you change branches in this guide.
> 
> I think that flow would break if we add a new submodule.
> 
> Pretty sure the `--init` is essentially free anyway.
> 
> I _do_ like the premise of this PR - when cloning add the extra argument. I _think_ `--recurse` may be sufficient vs `--recurse-submodules`... less typing?

Attempted to resolve this in efc85f176dd1638267812379e7ccb4ac5ce7e5ef.

Reply:
> The `--recurse` option is not available for me under the `git clone --help` listing.

It works, 'though :-)  But not fussed.